m4/lib-ld.m4

```diff
@@ -0,0 +1,119 @@
+# lib-ld.m4 serial 6
```

Comment:
sucking in get-text-devel bits means we have to track developments in libtool ourselves; this has bitten my butt in previous projects more often than tracking gettext has. Has this change been tested (with --disable-rpath active) on rhel8.x?

Reply:
Please test as you see fit. Absent this, we need to add gettext-devel  which is a bit odd IMO since that is a locale message translation service.